PyDSS/dssElement.py

```diff
@@ -138,14 +138,15 @@ def GetValue(self, VarName, convert=False):
             return None
         return VarValue
 
-
     def SetActiveObject(self):
         self._dssInstance.Circuit.SetActiveElement(self._FullName)
         if self._dssInstance.CktElement.Name() != self._dssInstance.Element.Name():
             raise InvalidParameter('Object is not a circuit element')
 
     def SetParameter(self, Param, Value):
         reply = self._dssInstance.utils.run_command(self._FullName + '.' + Param + ' = ' + str(Value))
+        if reply != "":
+            raise Exception(f"SetParameter failed: {reply}")
```

Comment:
Is this the correct thing to do? The existing code allows the caller to think this worked.

Reply:
I think I prefer the Exception. Something must be very wrong if this happens.